inference-engine/tests/functional/plugin/shared/src/subgraph_tests/basic_lstm.cpp

```diff
@@ -47,63 +47,75 @@ void Basic_LSTM_S::SetUp() {
     InferenceEngine::Precision netPrecision;
     std::tie(netPrecision, targetDevice, configuration) = this->GetParam();
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    hidden_size = 118;
+    outPrc = InferenceEngine::Precision::FP32;
 
-    auto params = ngraph::builder::makeParams(ngPrc, { {1, 490} });
+    function = GetNetwork(49, hidden_size, netPrecision, &hidden_memory_init, &cell_memory_init);
+}
 
-    hidden_size = 118;
-    const size_t batch_size = 1;
+std::shared_ptr<ngraph::Function> Basic_LSTM_S::GetNetwork(uint64_t thirdDimOut,
+    uint64_t hiddenSize,
+    const InferenceEngine::Precision& netPrecission,
+    std::vector<float>* hidden_memory_init_out,
+    std::vector<float>* cell_memory_init_out) {
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecission);
 
-    outPrc = InferenceEngine::Precision::FP32;
+    auto params = ngraph::builder::makeParams(ngPrc, { {1, 10 * thirdDimOut} });
 
-    //Reshape_1 [1,490] -> [1, 10, 49]
-    std::vector<uint64_t> outFormShapes1 = { batch_size, 10, 49 };
-    auto pattern1 = std::make_shared<ngraph::opset1::Constant>(ngraph::element::Type_t::i64, ngraph::Shape{3}, outFormShapes1);
+    const size_t batch_size = 1;
```

Comment:
I think, if the hidden size is defined in SetUp method, then, batch size should also be defined there to make it consistent.

Reply:
I have not changed that, and I don't think I should refactor existing code in this PR more than needed.